templates/data/postgresql/support.html

```diff
@@ -0,0 +1,635 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+https://docs.google.com/document/d/1scPktUiI2XRVoYcR4L6qUZ91iZ0upZA-FaqyJN7uMJg/edit
+{% endblock meta_copydoc %}
+
+{% block title %}Get comprehensive support for PostgreSQL, the world's most acclaimed open source database management system.{% endblock %}
```

Comment:
This title may change, it is longer than the 60 chars recommendation in the copy doc template ([comment ref](https://docs.google.com/document/d/1scPktUiI2XRVoYcR4L6qUZ91iZ0upZA-FaqyJN7uMJg/edit?disco=AAABT_iI3g4))